targets/psa_switch/pswitch_CLI.py

```diff
@@ -88,7 +88,43 @@ def main():
         args.thrift_ip, args.thrift_port, services
     )
 
-    runtime_CLI.load_json_config(standard_client, args.json)
+    json_ = runtime_CLI.load_json_config(standard_client, args.json)
```

Comment:
I think that's the idea, but as for the implementation would you mind just passing a function to `load_json_config` to perform the target-specific work, so that you don't end up repeating the `SUFFIX_LOOKUP_MAP` stuff below. Something like:
```python
def load_json_psa(json_dict):
    for j_meter in get_json_key("extern_instances"):
        # ...

def main():
    runtime_CLI.load_json_config(standard_client, args.json, load_json_psa)
    # ...
```
Then the runtime_CLI.py code will invoke `load_json_psa` at the right time. You should make this an optional parameter (and default it to `None`) so that you don't need to do any modification for simple_switch.

Let me know what you think

Reply:
This makes sense. I just made change accordingly. :D